examples/scripts/script_depsi_processing.py

```diff
@@ -42,6 +42,8 @@ def get_free_port():
 # Parameters
 method = 'nmad'  # Method for selection
 threshold = 0.45  # Threshold for selection
+dist_thres = 200  # Distance threshold for network selection
```

Comment:
I thought about this a little longer. I know that I proposed this, but I would like to change, since 'thres' suggests a maximum distance, instead of a minimum distance.

Therefore please change to 'min_dist'.